packer/conf/buildkite-agent/hooks/environment

```diff
@@ -8,6 +8,9 @@ source ~/cfn-env
 echo "~~~ :llama: Setting up elastic stack environment ($BUILDKITE_STACK_VERSION)"
 cat ~/cfn-env
 
+echo "Fixing permissions for '${BUILDKITE_AGENT_NAME}'..."
+sudo -E /usr/bin/fix-buildkite-agent-builds-permissions
```

Comment:
I think this also needs to remain in `environment` (i.e. not move to pre-checkout like #296) because otherwise I could do things like insert a plugin hook which changes BUILDKITE_CHECKOUT_PATH to something targeting /etc/passwd so it gets chowned and I can escalate out of my build.